Eatery/Views/Eateries/MealItemTableViewCell.swift

```diff
@@ -21,12 +29,59 @@ class MealItemTableViewCell: UITableViewCell {
         addSubview(nameLabel)
         nameLabel.translatesAutoresizingMaskIntoConstraints = false
         nameLabel.snp.makeConstraints { make in
-            make.edges.equalTo(self).inset(UIEdgeInsets(top: 8, left: 16, bottom: 8, right: 16))
+            make.edges.equalTo(self).inset(UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16))
         }
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    /// Configures the cell's nameLabel text to match the menu item text (adjusting for whether or not the item is healthy) and adds a favorite button to the cell corresponding to whether the user has favorited the menu item
+    func configure(for menuItem: Menu.Item) {
+        self.menuItem = menuItem
+        
+        let itemText = (menuItem.healthy) ? NSMutableAttributedString(string: "\(menuItem.name.trim()) ")
+        .appendImage(UIImage(named: "appleIcon")!, yOffset: -1.5) : NSMutableAttributedString(string: menuItem.name)
+        nameLabel.attributedText = itemText
+        
+        let starImageName = favoriteButtonAttributes!.imageName
+        let starImage = UIImage(named: starImageName)?.withRenderingMode(.alwaysTemplate)
+        favoriteButton = UIButton(type: .custom)
+        favoriteButton!.setImage(starImage, for: .normal)
+        favoriteButton!.tintColor = favoriteButtonAttributes!.tintColor
+        favoriteButton!.addTarget(self, action: #selector(tappedFavorite), for: .touchUpInside)
+        addSubview(favoriteButton!)
+        favoriteButton!.translatesAutoresizingMaskIntoConstraints = false
+        favoriteButton!.snp.makeConstraints { make in
+            make.top.bottom.equalToSuperview()
+            make.trailing.equalToSuperview().inset(16)
+            make.width.equalTo(favoriteButton!.snp_width)
+        }
+    }
```

Comment:
this function is called in `cellForRow` right? if so, we should move this to the init. all initialization of views should be inside the `init` and inside the `configure` we can update the views accordingly.

this should also allow you to make `favoriteButton` non-optional